pkg/cluster/k8sres.go

```diff
@@ -304,11 +300,15 @@ func (c *Cluster) generatePodTemplate(resourceRequirements *v1.ResourceRequireme
 	if c.OpConfig.WALES3Bucket != "" {
 		envVars = append(envVars, v1.EnvVar{Name: "WAL_S3_BUCKET", Value: c.OpConfig.WALES3Bucket})
 	}
+
+	if c.OpConfig.EtcdHost == "" {
```

Comment:
How is this handling not set? or null?

Reply:
it can't be null, only empty string
https://github.com/zalando-incubator/postgres-operator/blob/master/pkg/util/config/config.go#L50